src/plugins/foxPage/components/AssetActions.tsx

```diff
@@ -9,39 +11,77 @@ import {
   Tabs,
   Text as CText,
 } from '@chakra-ui/react'
+import { AssetId } from '@shapeshiftoss/caip'
+import { foxyAddresses } from '@shapeshiftoss/investor-foxy'
+import qs from 'qs'
 import { useTranslate } from 'react-polyglot'
+import { useHistory, useLocation } from 'react-router'
 import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
 import { Text } from 'components/Text/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useModal } from 'hooks/useModal/useModal'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { useGetAssetDescriptionQuery } from 'state/slices/assetsSlice/assetsSlice'
+import { selectAssetById } from 'state/slices/selectors'
+import { selectAccountIdsByAssetId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FoxAssetId } from '../constants'
+import { TrimDescriptionWithEllipsis } from '../utils'
 
 type FoxTabProps = {
-  assetIcon: string
-  assetSymbol: string
-  description: string
-  buyCTA: string
-  sellCTA: string
-  onReceiveClick: () => void
-  onBuyClick: () => void
+  assetId: AssetId
 }
 
-export const AssetActions = ({
-  assetIcon,
-  assetSymbol,
-  description,
-  buyCTA,
-  sellCTA,
-  onReceiveClick,
-  onBuyClick,
-}: FoxTabProps) => {
+const GetFoxyModalRoute = `/defi/token_staking/ShapeShift/overview`
+const GetFoxCoinbaseExternalUrl = `https://www.coinbase.com/price/fox-token`
```

Comment:
This one might stir a bit of controversy amongst reviewers since we have both an internal route and an external link handled the same way but IMO makes sense. The other way around would be a proper link (without an event handler) in one case, and no link on the DOM element, but an `onClick` with `history.push` in the case of the internal link, which might be quite the ugly logic